packages/fluentui/docs/src/prototypes/table/InteractiveTable.tsx

```diff
@@ -40,24 +53,30 @@ const InteractiveTable = () => {
           <Table.Cell content="Picture" key="pic" accessibility={gridHeaderCellBehavior} />
           <Table.Cell content="Age" key="age" accessibility={gridHeaderCellBehavior} />
         </Table.Row>
-        <Table.Row key="1" accessibility={gridRowBehavior}>
-          <Table.Cell content="1" key="1-1" accessibility={gridCellBehavior} />
-          <Table.Cell content="Roman van von der Longername" key="1-2" accessibility={gridCellBehavior} />
-          <Table.Cell content="None" key="1-3" accessibility={gridCellBehavior} />
-          <Table.Cell content="30 years" key="1-4" accessibility={gridCellBehavior} />
-        </Table.Row>
-        <Table.Row key="2" accessibility={gridRowBehavior}>
-          <Table.Cell content="2" key="1-1" accessibility={gridCellBehavior} />
-          <Table.Cell content="Alex" key="1-2" accessibility={gridCellBehavior} />
-          <Table.Cell content="None" key="1-3" accessibility={gridCellBehavior} />
-          <Table.Cell content="1 year" key="1-4" accessibility={gridCellBehavior} />
-        </Table.Row>
-        <Table.Row key="3" accessibility={gridRowBehavior}>
-          <Table.Cell content="3" key="1-1" accessibility={gridCellBehavior} />
-          <Table.Cell content="Ali" key="1-2" accessibility={gridCellBehavior} />
-          <Table.Cell content="None" key="1-3" accessibility={gridCellBehavior} />
-          <Table.Cell content="30000000000000 years" key="1-4" accessibility={gridCellBehavior} />
-        </Table.Row>
+        {withContextMenu(
+          <Table.Row key="1" accessibility={gridRowBehavior}>
+            <Table.Cell content="1" key="1-1" accessibility={gridCellBehavior} />
```

Comment:
```suggestion
            <Table.Cell content="1" accessibility={gridCellBehavior} />
```

`key` here and on all other `Table.Cell` and `Table.Roe` is redundant here

Reply:
fixed :)